.github/workflows/windows-ci.yml

```diff
@@ -6,12 +6,12 @@ on:
       os-version:
         description: 'OS version to run the workflow'
         required: false
-        default: 'windows-13'
+        default: 'windows-2022'
```

Comment:
Change the default also [here](https://github.com/eProsima/Fast-CDR/blob/0c21dd4a3508e7a6537c7f1046276f897fefda0f/.github/workflows/windows-ci.yml#L63)

Reply:
Addressed in 395884c